src/components/NewComps/Pricing/index.js

```
@@ -0,0 +1,241 @@
+   * @param {Object} plan - The plan object being evaluated.
+   * @param {Object} feature - The feature descriptor object which contains feature-related metadata.
+   * @returns {JSX.Element|null|any} The cell value for the given plan and feature, or an icon/component.
+   */
+  const getPlanInfoByFeature = (plan, feature) => {
+    if (!plan || !feature) return null;
+
+    if (Object.hasOwn(feature, "yearly")) {
+      return plan[tableType === "yearly" ? feature?.yearly : feature?.monthly];
+    } else if (feature.specificContent && !isFreePlan(plan)) {
+      return (
+        <span dangerouslySetInnerHTML={{ __html: feature.specificContent }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)